### DISCUSSION TITLE
Got the error while installing core emulator on Ubuntu

### DISCUSSION THREAD

**Discussion by meenakshi510** [coreemu/core#621] [Q&A] [answered]
I got the following error while installing the Core emulator on Ubuntu: 
Configure: error: please install python grpcio-tools

Please suggest a solution.

**Comment by bharnden** [comment, marked as answer]
There are the instructions here: https://coreemu.github.io/core/install.html

This appears like you may have tried just using `make` alone?

The error is pretty straight forward, but this is something that would be avoided by using the `./install.sh` command.

**Comment by meenakshi510** [comment]
@bharnden It is working. Thank you so much. I was using following commands:
$ git clone https://github.com/coreemu/core.git
$ cd core
$ ./bootstrap.sh
$ ./configure
$ make
$ sudo make install
